library/core/src/slice/mod.rs

```diff
@@ -1407,6 +1407,25 @@ impl<T> [T] {
         x.slice_contains(self)
     }
 
+    /// Returns `true` if the slice contains an element with the given value.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # #![feature(contains_ref)]
```

Comment:
nit: please leave the feature exposed, to make it easier for nightly users who want to use this to copy it.